src/main/java/io/appium/java_client/ios/IOSDeviceActionShortcuts.java

```diff
@@ -56,5 +57,14 @@ default void hideKeyboard(String strategy, String keyName) {
     default void shake() {
         CommandExecutionHelper.execute(this, shakeCommand());
     }
+    
+    /**
+     * Simulate touchId event
+     *
+     * @param match If true, simulates a successful fingerprint scan. If false, simulates a failed fingerprint scan
```

Comment:
If i'm not wrong, this will fail as per google style each doc line should end with a period.